docs/source/faq.md

```diff
@@ -78,11 +78,11 @@ for various features accordingly.
 If some of the modules UCX was built with are not found during runtime, they will
 be silently disabled.
 
-* **Basic shared memory and TCP support** - always enabled
+* **Basic shared memory and TCP support** - always enabled.
 * **Optimized shared memory** - requires knem or xpmem drivers. On modern kernels also CMA (cross-memory-attach) mechanism will be used.
```

Comment:
On modern kernels, CMA will also be used if available

Reply:
fixed